### PR TITLE
Task's update fix, empty context not permitted

### PR DESCRIPTION
**Pull request by slemrmartin**
Topological API's `body_params['context']` can't be empty hash for `permit` operation

### CODE REVIEW

**Comment by slemrmartin** [comment]
Fix of #47

File: lib/topological_inventory/ansible_tower/operations/core/service_order_mixin.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def order
             task_id, service_offering_id, service_plan_id, order_params = params.values_at(
               "task_id", "service_offering_id", "service_plan_id", "order_params")
 
-            update_task(task_id, :state => "running", :status => "ok", :context => {})
+            update_task(task_id, :state => "running", :status => "ok", :context => {:info => 'Task started'})
```
Comment (agrare) [inline lib/topological_inventory/ansible_tower/operations/core/service_order_mixin.rb]: @slemrmartin is context.info something that will actually be used or is this just to fix the error?
Could we just not pass any `:context` if it isn't going to contain a payload?

**Comment by agrare** [comment]
Oh I was thinking just don't send any context key at all, not sure if context => nil will work

**Comment by slemrmartin** [comment]
ok, it'll need different change, just a moment, :context is mandatory now

**Comment by agrare** [comment]
https://github.com/RedHatInsights/topological_inventory-ansible_tower/pull/50 should get this green

**Pull request merged**